pkg/features/featureflags.go

```diff
@@ -0,0 +1,114 @@
+package features
+
+import (
+	"github.com/golang/glog"
+)
+
+// Flags feature flags
+var Flags *Features
+
+func init() {
+	Flags = &Features{}
+}
+
+// Features ...
+type Features struct {
+	OC          OCFeatures
+	BC          BCFeatures
+	GM          GMFeatures
+	LogSeverity bool
+}
+
+// Print prints
+// out the internal values of feature gflags
+func (f Features) Print() {
+	f.OC.Print()
+	f.BC.Print()
+	f.GM.Print()
+	glog.Info("LogSeverity: ", f.LogSeverity)
+}
+
+// And applies a logical and on the feature sets
+func (f Features) And(other Features) *Features {
+	res := &Features{}
+	res.LogSeverity = f.LogSeverity && other.LogSeverity
+	res.OC = f.OC.And(other.OC)
+	res.BC = f.BC.And(other.BC)
+	res.GM = f.GM.And(other.GM)
+	return res
+}
+
+// OCFeatures ...
+type OCFeatures struct {
+	Enabled  bool
+	DualPort bool
+}
+
+// Print ...
+func (f OCFeatures) Print() {
+	glog.Info("OC Enabled: ", f.Enabled)
+	glog.Info("OC DualPort: ", f.DualPort)
+}
+
+// And applies a logical and on the feature sets
+func (f OCFeatures) And(other OCFeatures) OCFeatures {
+	res := OCFeatures{}
```

Comment:
small change using direct struct for easy readbility , probbaly do it for all .  Will make it easier to read 
return OCFeatures{
		Enabled:  f.Enabled && other.Enabled,
		DualPort: f.DualPort && other.DualPort,
	}